logos/bevy.svg

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Could you run this file through [`svgo`](https://github.com/svg/svgo) (web ui here: https://jakearchibald.github.io/svgomg/)